src/chrome/content/rules/Eustace.io.xml

```diff
@@ -1,6 +1,7 @@
 <!--
 	Nonfunctional hosts in *.eustace.io:
-		- www
+        - eustace.io
```

Comment:
Please use tabs to indent lines.

Reply:
Whoops. I apologize for all of the silly mistakes. Thank you for your patience.